content/projects/audi-tt/index.mdx

```diff
@@ -5,66 +5,88 @@ status: "in progress"
-</ ProjectHeader>
-<br />
+import Button from "../../../src/components/button";
+import TimelineItem from "../../../src/components/timeline-item";
+import Timeline from "../../../src/components/timeline";
+import ProjectPosts from "../../../src/components/project-posts";
+import AudiImage from "./audi-tt-2003.gif";
+import Project from "../../../src/components/project";
 
-<div style={{ marginLeft: `175px` }}>
-<Text sx={{ fontWeight: `medium`, fontSize: [3, 4], color: `heading` }}>
-  About
-</Text>
 
----
+<Project
+  headerImage={AudiImage}
+  title="Audi TT Restoration"
+  status="in progress"
+  buttons={[
+    {
+      text: "GitHub",
+      url: "https://github.com/mmcatcd",
+      color: "#242424",
+    },
+  ]}
+  postsFilter="tt"
+>
 
-<Text sx={{ fontWeight: `normal`, fontSize: [1, 2], color: `heading` }}>
-  The initial idea was to take use two scrapped cars to make a new and improved one. An electric car scrapped for structural damage could give cheap electronics and an ICE non-runner would provide a new home for the electronics. 
-</ Text>
-<br />
-<Text sx={{ fontWeight: `normal`, fontSize: [1, 2], color: `heading` }}>
-  Before we do this however, we want to start smaller. Let’s convert a Prius to all-electric. Along the way we have to get a Prius, do some mechanical work, convert to plug in hybrid, hack inverter and finally convert to full electric.
-</ Text>
+### About
+---
 
-</div>
-<br />
+Last year I bought a 2002 Audi TT, 225bhp with 151,000 miles for €400. Naturally, it needed some work.
+Initially, I thought it wouldn't take much to get the car back on the road. However, several trips to
+the garage later, I came to the realisation that in order for this car to become reliable again, I would
+need to do some restoration work.
 
-<div style={{ marginLeft: `175px` }}>
-<Text sx={{ fontWeight: `medium`, fontSize: [3, 4], color: `heading` }}>
-  Timeline
-</Text>
+I thought a Volkwagen Golf derived, "People's Car" would be an easy introduction to car restoration. 
+Boy was I wrong.
 
+### Timeline
 ---
-</div>
-
-<br />
-<Timeline title="Title" status="Completed" date="19.8.2020" type="normal">
-  Some sample description about what the shtory is like Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
-</Timeline>
-<Timeline title="In progress at the moment" status="In progress" type="normal">
-  Some sample description about what the shtory is like 
-</Timeline>
-<Timeline title="Not begun" status="Not started" type="normal">
-  Some sample description about what the shtory is like Lorem ipsum dolor sit amet,
-</Timeline>
-<Timeline title="Not begun, but last thing" status="Not started" type="end">
-  Some sample description about what the shtory is like Lorem ipsum dolor sit amet, consectetur 
-</Timeline>
 
-<br />
+  <Timeline>
```

Comment:
I also added a `Timeline` component that contains `TimelineItem` components. I guess this is more like how things are done in React and makes it a bit clearer what's going on here.

Reply:
I added a little bit to the timeline that allows you to add posts to each item too. This will have to be added